src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1339,22 +1724,61 @@ class ongoingTasksStats extends viewModelBase {
         const currentDatum = this.tooltip.datum();
 
         if (currentDatum !== element) {
-            const tooltipHtml = "Gap start time: " + (element).start.toLocaleTimeString() +
-                "<br/>Gap duration: " + generalUtils.formatMillis((element).durationInMillis);
+            const tooltipHtml = "Gap start time: " + element.start.toLocaleTimeString() +
+                "<br/>Gap duration: " + generalUtils.formatMillis(element.durationInMillis);
             this.handleTooltip(element, x, y, tooltipHtml);
         }
     }
 
+    private handleSubscriptionPendingTooltip(itemInfo: subscriptionPendingItemInfo, x: number, y: number) {
+        const currentDatum = this.tooltip.datum();
+
+        if (currentDatum !== itemInfo) {
+            let tooltipHtml = `<strong>*** ${itemInfo.title} ***</strong> <br/>`;
+            tooltipHtml += `Duration: ${generalUtils.formatMillis(itemInfo.duration)} <br/>`;
+            tooltipHtml += `<br>Client URI: <strong>${itemInfo.clientUri}</strong> <br/>`;
+            this.handleTooltip(itemInfo, x, y, tooltipHtml);
+        }
+    }
+
+    private handleSubscriptionConnectionTooltip(itemInfo: subscriptionConnectionItemInfo, x: number, y: number) {
+        const currentDatum = this.tooltip.datum();
+
+        if (currentDatum !== itemInfo) {
+            let tooltipHtml = `<strong>*** ${itemInfo.title} ***</strong> <br/>`;
+            tooltipHtml += `Duration: ${generalUtils.formatMillis(itemInfo.duration)} <br/>`;
+            tooltipHtml += `<br>Client URI: <strong>${itemInfo.clientUri}</strong> <br/><br/>`;
```

Comment:
not to many br's?

Reply:
I added the extra br's so that the Client Uri is more clear to see, otherwise info is too condensed.
However, I will remove them, and we need to reformat/rethink these tooltips format